### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by JimMadge** [alan-turing-institute/the-turing-way#3171]
Please leave this issue. I am hoping to use this as a place to automatically maintain a list of broken links.

**Comment by EstherPlomp** [comment]
I see a couple of Twitter links that time out but are not broken - not sure what to do with these in the sections that I'm more familiar with!

**Comment by JimMadge** [comment]
> I see a couple of Twitter links that time out but are not broken - not sure what to do with these in the sections that I'm more familiar with!

Yes there will be false negatives here. I expect the time out was because the link checker was sending requests to Twitter so often that some rate limiting was applied. I notice that there are no twitter timeouts after the report was updated this morning. There is a cache of results which is reused between runs, so it may not have needed to make so many requests to Twitter this time.

As a rule of thumb, 403 and timeout are quite likely to be false negatives where websites are interpreting requests as possible attacks or unwanted bots. In those cases, we can update the [configuration](https://github.com/alan-turing-institute/the-turing-way/blob/main/lychee.toml) to ignore particular urls or patterns.

404 and 50? are quite likely to be genuine broken links.

It would be good to add some guidance around this to the book.

**Comment by aleesteele** [comment]
Hi @JimMadge - Thanks so much for flagging these links for the community! 

It seems like fixing these links in different chapters could be a great first issue - but we might need to add a disclaimer to this issue, and/or create another means through which people could get involved with the project (and learn more about how to contribute more broadly in the process.

In general, I'm going to flag a "fixing links" section in #3319, as I agree that guidance on fixing these links is much-needed in the community handbook.

**Comment by JimMadge** [comment]
@aleesteele,

> It seems like fixing these links in different chapters could be a great first issue - but we might need to add a disclaimer to this issue, and/or create another means through which people could get involved with the project (and learn more about how to contribute more broadly in the process.

Yes absolutely!

There is some (technical) information [here](https://the-turing-way.netlify.app/community-handbook/infrastructure/infrastructure-external-link-check) about how the link check and workflow work.
However, information more like "what should I do with this" is missing.

That could be added to the [issue header](https://github.com/the-turing-way/the-turing-way/blob/main/.github/workflows/resources/external_link_check_header.md) so that it appears in this issue. Alternatively, it could be in the book?
